static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -282,6 +285,25 @@ const makeABTestVariant = (
             iframeId,
         },
 
+        canRun() {
```

Comment:
this used to be in `acquisitions-view-log` but was effectively just a variant-level `canRun()` check. so now that variants support `canRun()` functions, it is just that